lib/apiservers/engine/backends/container.go

```diff
@@ -267,7 +267,12 @@ func (c *Container) ContainerExecCreate(name string, config *types.ExecConfig) (
 		return nil
 	}
 
-	if err := retry.Do(operation, IsConflictError); err != nil {
+	// configure custom exec back off configure
+	backoffConf := retry.NewBackoffConfig()
```

Comment:
you have the same config duplicated twice - consider consolidating if they are _supposed_ to be the same.
Also note _why_ those times have been chosen; if lacking concrete numbers it's okay to express what you're attempting to accomplish with these timings. At the moment it's left to the imagination of the reader.